doc/api/events.md

```diff
@@ -1344,10 +1344,10 @@ This is not used in Node.js and is provided purely for completeness.
 added: v14.5.0
 -->
 
-* Type: {boolean} True for Node.js internal events, false otherwise.
+* Type: {boolean}
 
-Currently only `AbortSignal`s' `"abort"` event is fired with `isTrusted`
-set to `true`.
+The `AbortSignal` `"abort"` event is emitted with `isTrusted` set to `true`. The
```

Comment:
```suggestion
The {AbortSignal} `"abort"` event is emitted with `isTrusted` set to `true`. The
```
nit: I think it would be helpful to turn this into a link, like it is done everywhere this occurs in the file :)